src/humanize/number.py

```diff
@@ -205,23 +205,44 @@ def intword(value: NumberOrString, format: str = "%.1f") -> str:
     except (TypeError, ValueError):
         return str(value)
 
+    is_negative = value < 0
+
+    if is_negative:
+        value *= -1
```

Comment:
How about something like this?

```suggestion
    if value < 0:
        value *= -1
        negative_prefix = "-"
    else:
        negative_prefix = ""
```

Then we can remove the `if is_negative` bits (and some duplication) and always prepend `negative_prefix` to the return?

Reply:
@hugovk much cleaner! thanks for the suggestion